mlpipeline/mlpipeline/app.py

```
@@ -0,0 +1,85 @@
+
+# DATABASE CONFIG
+DB_DSN = os.getenv("DATABASE_URL", "postgresql://user:password@localhost:5432/mydb")
+
+@asynccontextmanager
+async def lifespan():
+    print("Startup: Loading resources...")
+
+    try:
+        ml_models["embedding_model"] = SentenceTransformer('sentence-transformers/all-MiniLM-L6-v2')
+        print("AI Model Ready!")
+    except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)